tests/e2e/default/enketo/unauthorized-form.wdio-spec.js

```diff
@@ -38,9 +38,7 @@ describe('Unauthorized form', () => {
   afterEach(async () => await utils.revertSettings(true));
 
   it('should display unauthorized error message in reports tab when form expression does not match', async () => {
-    await commonPage.goToReports();
-
-    await browser.url('#/reports/add/pregnancy');
+    await browser.url('/#/reports/add/pregnancy');
```

Comment:
Why we don't use here the new commonPage.goToUrl() method?

Reply:
That is a very good question. I'm not the original author, but it seems like `commonPage.goToUrl()` should work here.  
Do you think we should encourage the usage of `commonPage.goToUrl()` always as a replacement for `browser.url(..`? 
If yes, I can open a separate PR to do the replacement all over the e2e, so that's the way to go for other developers when writing tests.

Reply:
It is a great idea to do this.